Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    enumbler (0.6.11)
+    enumbler (0.6.12)
```

Comment:
I bumped again to `.13` because this was released early.  Whoops.  

I would say when you are satisfied with it we should move to `0.7.0` beacuse we are changing functionality ... even though this is still "pre-release" is a breaking change.